src/app/inventory/d2-season-info.ts

```diff
@@ -0,0 +1,78 @@
+// TODO: Update on season change
+export const D2CurrentSeason: number = 5;
```

Comment:
Using the enum below, could you do `const D2CurrentSeason: number = D2SeasonEnum.BLACK_ARMORY;` ?